.github/workflows/.travis.yml

```diff
@@ -0,0 +1,42 @@
+name: ".travis.yml"
+
+on:
+    push:
+        branches: [ master ]
+    pull_request:
+        branches: [ master ]
+
+jobs:
+    build:
+        runs-on: ${{ matrix.os }}
+        env: 
+            TOXENV: ${{ matrix.test }}
+            RUN_INTEGRATION_TESTS: ${{ matrix.test_number}}
+        steps:
+            - uses: actions/checkout@v3
+            - run: pip install "tox"
+            - run: pip install bandit
+            - run: pip install codecov
+            - run: pip install slugs
+            - run: sudo chmod -R 777 /usr/local/lib/
+            - run: python3 setup.py install
+            - run: RUN_INTEGRATION_TESTS=${{ matrix.test_number }} ./.travis/run.sh
+            - run: codecov
+        strategy:
+            matrix:
+                test: ["pep8", "bandit", "docs"]
+                test_number: ["0","1"]
```

Comment:
The test_number doesn't affect the pep8, bandit, or docs tests, so we're running each of those tests twice for no reason.
It would be preferable to just remove this line, but I'm not sure if that would break the env section above.
If the tests break when `test_number` is undefined, then we should change this list to just have one entry.